### PR TITLE
add javascript submodules

### PR DESCRIPTION
**Pull request by kiyanov**
Here's a list of valuable resources that I came across during my research. Please find them below:

### CODE REVIEW

File: javascript/Front-end-Developer-Interview-Questions
Comment (chriseugenerodriguez) [inline javascript/Front-end-Developer-Interview-Questions:1]: This doesnt have any material I can use. its just a list of questions.

**Review by chriseugenerodriguez — COMMENTED** [review]
These repos should just have javascript in them.

File: javascript/Front-end-Developer-Interview-Questions
Comment (kiyanov) [reply javascript/Front-end-Developer-Interview-Questions:1]: You are right, removed

File: javascript/interactive-coding-challenges
Comment (chriseugenerodriguez) [inline javascript/interactive-coding-challenges:1]: this is for python.

File: javascript/Awesome-JavaScript-Interviews
Comment (chriseugenerodriguez) [inline javascript/Awesome-JavaScript-Interviews:1]: this doesnt have link for me to review.

**Review by chriseugenerodriguez — COMMENTED** [review]
please review comments.

File: javascript/Awesome-JavaScript-Interviews
Comment (kiyanov) [reply javascript/Awesome-JavaScript-Interviews:1]: Uplease re-check, it should be this repo https://github.com/rohan-paul/Awesome-JavaScript-Interviews#readme

File: javascript/interactive-coding-challenges
Comment (kiyanov) [reply javascript/interactive-coding-challenges:1]: removed

File: javascript/Awesome-JavaScript-Interviews
Comment (chriseugenerodriguez) [reply javascript/Awesome-JavaScript-Interviews:1]: [Awesome-JavaScript-Interviews](https://github.com/rohan-paul/Awesome-JavaScript-Interviews) has a bunch of different topics, for next iteration were going to take all the code out.

**Review by chriseugenerodriguez — APPROVED** [review]

**Pull request merged**